tests/trace/test_evaluations.py

```diff
@@ -973,3 +973,37 @@ def score(text, model_output) -> bool:
             id=list(score.calls())[0].id,
         ).uri()
     )
+
+
+@pytest.mark.asyncio
+async def test_feedback_is_correctly_linked_with_scorer_subclass(client):
+    @weave.op
+    def predict(text: str) -> str:
+        return text
+
+    class MyScorer(Scorer):
+        @weave.op()
+        def score(self, text, model_output) -> bool:
+            return text == model_output
+
+    scorer = MyScorer()
+    eval = weave.Evaluation(
+        dataset=[{"text": "hello"}],
+        scorers=[scorer],
+    )
+    res = await eval.evaluate(predict)
+    calls = client.server.calls_query(
```

Comment:
Do you need to go down to this level?  The test might be more appropriate with `client.calls()`

Reply:
I really don't like the calls API - especially how it gets feedback. This is much better for asserting the data I am interested in asserting

Reply:
My preference is to fix the calls API.  It feels wrong to reach into the server here.  What's wrong with how it gets feedback?

Reply:
I do this in countless tests right now. all i want is to assert the raw data, not worry aobut the higher level calls iterator or opaque feedback iterator stuff

Reply:
Not friendly: 

```
    @property
    def feedback(self) -> RefFeedbackQuery:
        if not self.id:
            raise ValueError("Can't get feedback for call without ID")
        if self._feedback is None:
            project_parts = self.project_id.split("/")
            if len(project_parts) != 2:
                raise ValueError(f"Invalid project_id: {self.project_id}")
            entity, project = project_parts
            weave_ref = CallRef(entity, project, self.id)
            self._feedback = RefFeedbackQuery(weave_ref.uri())
        return self._feedback

```

Reply:
fwiw, i try to avoid using the WeaveClient directly unless i am explicitly trying to test functionality of the WeaveClient. I find it much more cumbersome

Reply:
I won't block on this, but I think reaching into the server is not ideal.  All of this code is at the "client level", so the checks should also be at the "client level".  If the client sucks we should fix it